crates/sc-proof-of-time/src/gossip.rs

```diff
@@ -95,22 +132,102 @@ impl<Block: BlockT> PotGossip<Block> {
             }
         }
     }
+
+    /// Initializes the chain state from the consensus tip info.
+    async fn initialize(&self) {
```

Comment:
I am not sure about initialize() happening both from the time keeper and the gossip components, particularly in the context of the node restart, etc that we talked about (this may be ok, but not clear right now). After 1860, 1861 the init path would be simplified, so it probably is fine then

Reply:
Upcoming PR actually removed all initialization of gossip, I think I'll have to change it since initialization changes there as well.